Ziti.NET.Standard/src/OpenZiti/ZitiService.cs

```diff
@@ -166,12 +166,7 @@ private void native_on_client_cb(IntPtr ziti_connection_server, IntPtr ziti_conn
             if (nativeService.posture_query_map != IntPtr.Zero) {
 	            model_map_impl impl = Marshal.PtrToStructure<model_map_impl>(nativeService.posture_query_map);
 
-                List<model_map_entry> nativeModelMapList;
-                if (impl.size.val > 1) {
-                    nativeModelMapList = MarshalUtils<model_map_entry>.convertPointerToList(impl.entries);
-                } else {
-                    nativeModelMapList = new List<model_map_entry>() { Marshal.PtrToStructure<model_map_entry>(impl.entries) };
-                }
+                List<model_map_entry> nativeModelMapList = MarshalUtils<model_map_entry>.convertPointerMapToList(impl.entries);

```

Comment:
Not on this PR but fyi... We are going to have to rework this. None of these structs are 'public'.  not a problem for now but soon.